apps/roam/src/components/DiscourseNodeSearchMenu.tsx

```diff
@@ -520,4 +526,38 @@ export const renderDiscourseNodeSearchMenu = (props: Props) => {
   );
 };
 
+export const NodeSearchMenuTriggerSetting = ({
+  onloadArgs,
+}: {
+  onloadArgs: OnloadArgs;
+}) => {
+  const extensionAPI = onloadArgs.extensionAPI;
+  const [nodeSearchTrigger, setNodeSearchTrigger] = useState<string>(
+    extensionAPI.settings.get("node-search-trigger") as string,
+  );
+
+  const handleNodeSearchTriggerChange = (
+    e: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    const value = e.target.value.trim();
+    const trigger = value
+      .replace(/"/g, "")
+      .replace(/\\/g, "\\\\")
+      .replace(/\+/g, "\\+")
+      .trim();
+
+    setNodeSearchTrigger(trigger);
+    extensionAPI.settings.set("node-search-trigger", trigger);
+  };
+  return (
+    <InputGroup
+      value={nodeSearchTrigger}
+      onChange={handleNodeSearchTriggerChange}
+      placeholder="@"
```

Comment:
As we are not currently setting a default, I think this would be confusing to the user.  Let's change it to "Click to set trigger" to match
![image](https://github.com/user-attachments/assets/420cea84-8c94-4eaf-a8ac-a4ef6ae1a85c)